src/codec/RowWriterV2.h

```diff
@@ -254,7 +254,7 @@ class RowWriterV2 {
   WriteResult write(ssize_t index, uint64_t v);
 
   WriteResult write(ssize_t index, const std::string& v);
-  WriteResult write(ssize_t index, folly::StringPiece v);
+  WriteResult write(ssize_t index, folly::StringPiece v, bool isWKB = false);
```

Comment:
I think you could add a new structure like `struct WKB {folly::StringPiece}`.

Reply:
I want to reuse the code of `write(ssize_t index, folly::StringPiece v)`